google-cloud-filestore-v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "file.googleapis.com",
+    "client_documentation": "https://googleapis.dev/ruby/google-cloud-filestore-v1/latest",
+    "distribution_name": "google-cloud-filestore-v1",
+    "language": "ruby",
+    "name": "file",
+    "name_pretty": "Filestore V1 API",
+    "product_documentation": "https://cloud.google.com/filestore/",
+    "repo": "googleapis/google-cloud-ruby",
+    "requires_billing": true,
```

Comment:
Is it OK that this library doesn't have `ruby-cloud-env-prefix`?

Reply:
I've now stopped adding the prefix for all new libraries. We're fast polluting the environment variable namespace, and increasing the possibility that one of the environment variable names we read will collide with a user's application env variable, resulting in us reading/using the wrong credentials. I think it was a mistake to provide that prefix for any libraries beyond our often-used veneers.